motion_predict/include/motion_predict/predict_ctrv.hpp

```diff
@@ -32,7 +32,8 @@ namespace ctrv
  * \param  delta_t prediction time step size in seconds
  * \param  period The total prediction period in seconds
  * \param  process_noise_max is the maximum process noise of the system
- *
+ * \note twist.linear vector is expected to be in the map frame and used as the yaw for future prediction.
```

Comment:
Same comments on function comments here as in `motion_predict.hpp`

Reply:
ditto